content/document.js

```diff
@@ -213,9 +213,29 @@ const findByURL = memoize((url, fields = null) => {
   return document ? { contentRoot: CONTENT_ROOT, folder, document } : null;
 });
 
-function findAll() {
+function findAll(
+  { files, folderSearch } = { files: new Set(), folderSearch: null }
+) {
+  if (!files instanceof Set) throw new Error("TypeError: 'files' not a Set");
```

Comment:
oh another nit: why not throw a [real TypeError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError/TypeError)?

Reply:
done.